### DISCUSSION TITLE
Wrong type property of isOn it should be isClose based on documentation of johnny-five

### DISCUSSION THREAD

**Discussion by matigarowel** [DefinitelyTyped/DefinitelyTyped#57838] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9035bba65ea608e5ac12f7f23b8b6a07e600865f/types/johnny-five/index.d.ts#L713

**Comment by martin-badin** [comment]
Can you provide documentation?

**Reply by matigarowel** [reply]
Hi Im using npm @types/johnny-five, there is missed aligned variable.
If I use isOn it doesn't work.
This is the line
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9035bba65ea608e5ac12f7f23b8b6a07e600865f/types/johnny-five/index.d.ts#L713

**Reply by martin-badin** [reply]
Please @matigarowel could you provide documentation?

[a comment by an automated account is not shown]
